tests/test_billboard.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+from mirdata.datasets import billboard
+from mirdata import annotations
+
+
+def test_track():
+
+    default_trackid = "3"
+    data_home = "tests/resources/mir_datasets/billboard"
+    dataset = billboard.Dataset(data_home)
+    track = dataset.track(default_trackid)
+
+    # test attributes are loaded as expected
+    assert track.track_id == default_trackid
+    assert track._data_home == data_home
+
+    assert track._track_paths == {
+        "audio": [
+            "audio/1960s/James Brown/I Don't Mind/audio.flac",
+            "bb9f022b25c43983cf19aef562b00eac",
+        ],
+        "salami": [
+            "McGill-Billboard/0003/salami_chords.txt",
+            "8deb413e4cecadcffa5a7180a5f4c597",
+        ],
+        "bothchroma": [
+            "McGill-Billboard/0003/bothchroma.csv",
+            "c92ee46045f5bacd681543e8b9aa55b8",
+        ],
+        "tuning": [
+            "McGill-Billboard/0003/tuning.csv",
+            "31c744b447b739bc8c4ed29891dc1fb1",
+        ],
+        "lab_full": [
+            "McGill-Billboard/0003/full.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmin7": [
+            "McGill-Billboard/0003/majmin7.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmin7inv": [
+            "McGill-Billboard/0003/majmin7inv.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmin": [
+            "McGill-Billboard/0003/majmin.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+        "lab_majmininv": [
+            "McGill-Billboard/0003/majmininv.lab",
+            "59c73209de645ef7e4e4293f4d6882b3",
+        ],
+    }
+
+    assert track.title == "I Don't Mind"
+    assert track.artist == "James Brown"
+
+    # test that cached properties don't fail and have the expected type
+    assert type(track.sections) is annotations.SectionData
```

Comment:
Please add the `chord` properties here as well

Reply:
Fixed in https://github.com/Spijkervet/mirdata/commit/f33280b69f2341bde2d79e847720a4bcb6e5037c